start-otd.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Check if OpenTabletDriver.Daemon is running
+if ! ps ax | grep -v grep | grep "OpenTabletDriver.Daemon" > /dev/null
+then
+    echo "OpenTabletDriver.Daemon is not running, starting it now..."
+    # Start Daemon from the current directory
+    /app/bin/OpenTabletDriver.Daemon &
```

Comment:
flaptak is not really designed to run daemons

Reply:
Is this allowed on flathub? I know it's not a good idea to run daemon inside flatpak, but I it actually works.